overview/default.nix

```diff
@@ -148,11 +148,25 @@ let
         '';
     };
 
+    subgrants = rec {
+      one = subgrant: ''
+        <li>
+          <https://nlnet.nl/project/${subgrant}>
+        </li>
+      '';
+      many =
+        subgrants:
+        optionalString (subgrants != [ ]) ''
+          <ul>
+            ${concatLines (map one subgrants)}
+          </ul>
+        '';
+    };
+
     projects = {
       one = name: project: ''
         ${heading 1 name}
-        <https://nlnet.nl/project/${name}>
-
+        ${render.subgrants.many (project.metadata.subgrants or [ ])}
```

Comment:
Would it be feasible to fall back to using `<https://nlnet.nl/project/${name}>`  if `project.metadata.subgrants` doesn't exist? As such, we won't break the links for the old projects.

Reply:
I think it's okay to break it temporarily (since it's a partial break) and in one of the next steps to invert the compatibility wrapper so `//projects-old` are converted to the new format.